src/customRules/importTypeOrderRule.ts

```diff
@@ -0,0 +1,68 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+
+export class Rule extends Lint.Rules.AbstractRule {
+  /* tslint:disable:object-literal-sort-keys */
+  public static metadata: Lint.IRuleMetadata = {
+    'ruleName': 'import-type-order',
+    'description': 'Enforce structure to your imports. Import structure should be listed in the following order: modules, absolute imports,  relative parent/ancestor directories, relative sibling directors.',
```

Comment:
`Improves readability and organization by grouping related imports together.  Imports should be listed in order of: external modules, absolute paths, relative paths, relative siblings.`